Final_Project_Paper-AR.qmd

```diff
@@ -0,0 +1,174 @@
+
+```{r}
+#Load in Libraries
+library(tidytext)
+library(tidyverse)
+library(readtext)
+library(tm)
+library(topicmodels)
+library(widyr)
+library(SnowballC)
+library(DigitalMethodsData)
+library(wordVectors)
+library(magrittr)
+library(dplyr)
+library(ggplot2)
+```
+
+```{r}
+#load in dataset
+metadata <- read.csv("ForeignPolicyBulletinData/metadata.csv")
+
+fp.metadata <- separate(metadata,
+         col = date,
+         into = c("year", "month", "day"),
+          remove = FALSE)
+
+fpmeta <- as.data.frame(fp.metadata)
+fp_texts <- readtext(paste("ForeignPolicyBulletinData/txt/", "*.txt", sep=""))
+fp_whole <- full_join(fpmeta, fp_texts, by = c("filename" = "doc_id")) %>% as_tibble() 
+
+stop_words_custom <- stop_words %>% 
+  add_row(word="foreign", lexicon="NA")%>%
+  add_row(word="policy", lexicon="NA")%>%
+  add_row(word="international", lexicon="NA")%>%
+  add_row(word="york", lexicon="NA")%>%
+  add_row(word="act", lexicon="NA")%>%
+  add_row(word="bulletin", lexicon="NA")%>%
+  add_row(word="3", lexicon="NA")%>%
+  add_row(word="association", lexicon="NA")%>%
+  add_row(word="class", lexicon="NA")%>%
+  add_row(word="published", lexicon="NA")%>%
+  add_row(word="2", lexicon="NA")%>%
+  add_row(word="december", lexicon="NA")%>%
+  add_row(word="march", lexicon="NA")%>%
+  add_row(word="post", lexicon="NA")%>%
+  add_row(word="vol", lexicon="NA")%>%
+  add_row(word="membership", lexicon="NA")%>%
+  add_row(word="incorporated", lexicon="NA")%>%
+  add_row(word="weekly", lexicon="NA")%>%
+  add_row(word="5", lexicon="NA")%>%
+  add_row(word="1", lexicon="NA")%>%
+  add_row(word="4", lexicon="NA")%>%
+  add_row(word="22", lexicon="NA")%>%
+  add_row(word="8", lexicon="NA")%>%
+  add_row(word="6", lexicon="NA")%>%
+  add_row(word="7", lexicon="NA")%>%
+  add_row(word="ing", lexicon="NA")%>%
+  add_row(word="de", lexicon="NA")%>%
+  add_row(word="tion", lexicon="NA")%>%
+  add_row(word="vera", lexicon="NA")%>%
+  add_row(word="vou", lexicon="NA")%>%
+  add_row(word="editor", lexicon="NA")%>%
+  add_row(word="subscription", lexicon="NA")%>%
+  add_row(word="page", lexicon="NA")%>%
+  add_row(word="printed", lexicon="NA")%>%
+  add_row(word="ment", lexicon="NA")%>%
+  add_row(word="matter", lexicon="NA")%>%
+  add_row(word="publications", lexicon="NA")%>%
+  add_row(word="con", lexicon="NA")%>%
+  add_row(word="street", lexicon="NA")
+
+tidy_fp <- fp_whole %>%
+  unnest_tokens(word, text) %>% 
+  filter(str_detect(word, "[a-z']$")) %>% 
+  anti_join(stop_words_custom)
+
+tidy_fp <- tidy_fp %>% filter(!grepl('[0-9]', word))
```

Comment:
Another code block that doesn't drive your narrative forward and could be hidden